docs/install-and-run.md

```diff
@@ -263,40 +275,39 @@ your test and in your application code and use the built-in tools seamlessly.
 
 ### Swift
 
-If you want to write your test cases in Swift, you need to perform two additional
-steps in your setup:
+To use EarlGrey with Swift, include the `EarlGrey.swift` file in your Swift Test Project. The file
+contains functions that implement C preprocessor macros that aren't available in Swift. There are
+two ways to do so:
 
-1. Include the `EarlGrey.swift` file in your Swift Test Project. The file contains functions that
-   implement C preprocessor macros that aren't available in Swift.
+#### Using the [earlgrey gem](https://rubygems.org/gems/earlgrey): 
+* We recommend uisng the EarlGrey gem to use EarlGrey with Swift.
 
-  * **If you're using the [earlgrey gem](https://rubygems.org/gems/earlgrey)**
+The EarlGrey gem also copies the `EarlGrey.swift` file to your project directory if you're
+passing the `{swift: true}` parameter to the args for the `configure_for_earlgrey()` method as
+in the below snippet. This is compiled into the tests along with EarlGrey.
 
-      The EarlGrey gem also copies the `EarlGrey.swift` file to your project directory if you're passing the
-      `{swift: true}` parameter to the args for the `configure_for_earlgrey()` method as in the below snippet.
-      You need to manually link this file to your project.
-
-      ```ruby
-      configure_for_earlgrey(installer, PROJECT_NAME, test_target, test_target + '.xcscheme', {swift: true})
-      ```
+```ruby
+configure_for_earlgrey(installer, PROJECT_NAME, test_target, test_target + '.xcscheme', {swift: true})
+```
 
-      EarlGrey currently supports Swift 3.0 and 2.3. While 3.0 is the default, you can also use 2.3 by adding
-      `{swift: true, swift_version: 2.3}`.
+EarlGrey currently supports Swift 3.0 and 2.3. While 3.0 is the default, you can also use 2.3
+by adding `{swift: true, swift_version: 2.3}`.
```

Comment:
Removed this altogether. Don't think it's needed and the gem docs provide any needed details.